ai_ta_backend/vector_database.py

```diff
@@ -293,24 +293,29 @@ def bulk_ingest(self, s3_paths: Union[List[str], str], course_name: str, **kwarg
 
   def _ingest_single_py(self, s3_path: str, course_name: str):
     try:
-      with NamedTemporaryFile() as tmpfile:
-        # download from S3 into vtt_tmpfile
-        self.s3_client.download_fileobj(Bucket=os.getenv('S3_BUCKET_NAME'), Key=s3_path, Fileobj=tmpfile)
-        loader = PythonLoader(tmpfile.name)
-        documents = loader.load()
-        texts = [doc.page_content for doc in documents]
-        metadatas: List[Dict[str, Any]] = [{
+      print("in ingest_py")
+      file_name = s3_path.split("/")[-1]
+      file_path = "media/" + file_name

```

Comment:
Sorry I'm confused about this line. Why is `s3_path` always in `media/`? Is that a hack from testing?

Reply:
I am downloading and storing the python files to the media folder for processing and then deleting them at the end. Should I create another a dedicated folder for that?

Reply:
sounds good